test/e2e-tests.sh

```diff
@@ -48,7 +48,8 @@ function knative_setup() {
 
 # Script entry point.
 
-initialize $@
+# Istio is installed as a add-on with cluster by default. Skip installing it again.
+initialize $@ --skip-istio
```

Comment:
We don't want this in Serving.

Reply:
`--skip-istio` flag is for skipping Istio Addon

Reply:
serving tests installs istio separately. this flag will skip installing it with the add-on

Reply:
updated name to skip-istio-addon and the comment to be more clear.